app/assets/stylesheets/gov.scss

```diff
@@ -17,10 +17,22 @@
 
 .govuk-grid-row .govuk-grid-column-full .govuk-grid-row .govuk-grid-column-one-half:nth-child(odd),
 .govuk-grid-row .govuk-grid-column-two-thirds .govuk-grid-row .govuk-grid-column-one-half:nth-child(odd) {
-  padding: 0 15px 0 0;
+  padding: 0;
```

Comment:
you can nest media queries within here which makes it a bit nicer and less duplication, for example: 

```scss
.header {
	background: $black;

	@media only screen and (min-width: 640px) {
		background: $white;
	}
}
```

Reply:
Cool! I didn't know this. Definitely better. I will refactor to this.

Reply:
Updated in https://github.com/DFE-Digital/single-place-for-education-frontend/pull/167/commits/7aed41296d5c9dc4b78f38769d92a21aa653b5c7.